src/NServiceBus.Core/ISendOnlyBus.cs

```diff
@@ -37,5 +37,21 @@ public interface ISendOnlyBus : IDisposable
         /// <param name="messageConstructor">An action which initializes properties of the message.</param>
         /// <param name="options">The options for the send.</param>
         Task SendAsync<T>(Action<T> messageConstructor, SendOptions options);
+
+        /// <summary>
+        /// Subscribes to receive published messages of the specified type.
+        /// This method is only necessary if you turned off auto-subscribe.
+        /// </summary>
+        /// <param name="eventType">The type of event to subscribe to.</param>
+        /// <param name="options">Options for the subscribe.</param>
+        Task SubscribeAsync(Type eventType, SubscribeOptions options);
```

Comment:
within the message handlers, yes. But it's still the interface to use on the "non-context-aware" parts of an application, right?